satpy/composites/__init__.py

```diff
@@ -743,6 +744,178 @@ def __call__(self, projectables, optional_datasets=None, **info):
         return proj
 
 
+class ParallaxCorrector(CompositeBase):
+    """Parallax Corection
+
+    Ref:
+        code: https://github.com/ORAC-CC/orac/blob/master/post_processing/correct_parallax.F90
+        paper: 10.3390/rs12030365 (doi)
+    """
+
+    def _parallax_corr(self, optional_datasets,
+                       sat_lon, sat_lat, sat_h,
+                       lon, lat):
+        """Applying Parallax Correction"""
+        # varius earth radius information
+        radius_eq = 6378.077
+        radius_pole = 6356.577
+        radius_ratio = radius_eq/radius_pole
+
+        # angle conversion to radians
+        sat_lat = np.deg2rad(sat_lat)
+        sat_lon = np.deg2rad(sat_lon)
+        lat = np.deg2rad(lat)
+        lon = np.deg2rad(lon)
+
+        # cartesian coordinates for the satellite
+        # satlat_geod is the geodetic satellite latitude
+        sat_lat_geod = arctan(tan(sat_lat)*radius_ratio**2)
```

Comment:
In my code I'm dividing by `radius_ratio` rather than multiplying. Not sure who is right here. ;)
If you notice errors, try changing this to a division.